src/index.html

```diff
@@ -2,15 +2,47 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      class="form" 
+      action="#" 
+      method="post"
+      data-qa="big"
+    >
+      <div class="form__field">
+        <label class="form__label form__label--large">
+          <input 
+            class="form__search form__search--large" 
+            type="text" 
+            name="searchQuery"
+            placeholder="Try &quot;Los Angeles&quot;"
+            data-qa="keypress"
+          >
+        </label>
+      </div>
+    </form>
+    <form 
+      class="form" 
+      action="#" 
+      method="post"
+      data-qa="small"
+    >
+      <div class="form__field">
+        <label class="form__label form__label--small">
+          <input 
+            class="form__search form__search--small" 
+            type="text" 
+            name="searchQuery"
+            placeholder="Try &quot;Los Angeles&quot;"
+          >
+        </label>
+      </div>
+    </form>
   </body>
-</html>
+</html>
```

Comment:
add new line in the end of the line

Reply:
Hi! As I understood - after </html> should be empty line? Why is it necessary?

Reply:
> add new line in the end of the line

Hi! As I understood - after should be empty line? Why is it necessary?

Reply:
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file